Makefile

```diff
@@ -568,7 +568,7 @@ lint: cpplint rcpplint jnilint pylint
 
 cpplint:
 	3rdparty/dmlc-core/scripts/lint.py mxnet cpp include src plugin cpp-package tests \
-	--exclude_path src/operator/contrib/ctc_include
+	--exclude_path src/operator/contrib/ctc_include src/operator/nn/ctc_include
```

Comment:
Hi @apeforest, I think if we are adding it to src/operator/nn we should enforce the same lint checks enforced on rest of the code of MXNet. We can create exception for contrib but not very comfortable adding exception for src/operator/nn dir. I would like to hear other opinions.

Reply:
@anirudh2290 ctc_include is a 3rd party library and I plan to move it from src/operator/contrib/ctc_include to src/operator/nn/ctc_include in my PR #12637. I need this cpp lint to be excluded from Makefile before CI can pass. Thanks

Reply:
i understand that but i was concerned about adding 3rdparty library in src/operator dir when it doesnt conform to mxnet style guidelines. Also, what if more files were to be added to this dir in the future, which don't adhere to lint and not caught in review.

Reply:
Hi @szha, what's the guideline of moving an operator from contrib to regular and its 3rd party dependency? Thanks!